android/version.gradle

```diff
@@ -0,0 +1,48 @@
+def String generateVersionName() {
```

Comment:
Mmm well... This is not Fastlane. This file is how Wolox handles the version name and code and is out of the Fastlane scope.
I'd extract it to another repository or gist, and then in the `build.gradle` file you can use using `apply plugin: "https://urlofthefile.com/tuvi"